restapi.go

```diff
@@ -1228,6 +1228,28 @@ func (s *Session) ChannelMessageSendTTS(channelID string, content string) (st *M
 	return s.channelMessageSend(channelID, content, true)
 }
 
+// ChannelMessageSendEmbed sends a message to the given channel with embedded data (bot only).
+// channelID : The ID of a Channel.
+// embed     : The embed data to send.
+func (s *Session) ChannelMessageSendEmbed(channelID string, embed *MessageEmbed) (st *Message, err error) {
```

Comment:
Should not it be ``ChannelMessageWithEmbedSend`` or ``ChannelMessageEmbedSend`` or similar?
Sorry about commenting on a merged pull request. Just something I noticed.